src/Controllers/ItemController.php

```diff
@@ -96,4 +122,23 @@ public function showItemOld($name = null, $itemId = null)
 
         return $this->showItem("", (int)$itemId, 0);
     }
+
+    public function showItemForCategory($category = null)
+    {
+
+        /** @var ItemListService $itemListService */
+        $itemListService = pluginApp(ItemListService::class);
+        $itemList = $itemListService->getItemList(ItemListService::TYPE_CATEGORY, $category->id, null, 1);
+        if (!count($itemList['documents']))
+        {
+            $itemList = $itemListService->getItemList(ItemListService::TYPE_RANDOM);
```

Comment:
Der Teil mit dem TYPE_RANDOM sorgt dafür, dass im TWIG eine falsche Kategorie ID ankommt. In meinem Fall die ID 17